x_axis:
  Kp: -0.15
  Kd: -0.15
  Ki: 0.0

y_axis:
  Kp: -0.15
  Kd: -0.15
  Ki: 0.0

z_axis:
  Kp: 1
  Kd: -0.00
  Ki: 0.0

yaw_axis:
  Kp: 0.0
  Kd: -0.00
  Ki: 0.0
